src/pages/GetStarted/CustomWorkspaceTab/DevfileSelector.tsx

```diff
@@ -57,41 +63,52 @@ export class DevfileSelectorFormGroup extends React.PureComponent<Props, State>
     this.devfileLocationRef = React.createRef();
   }
 
-  private onDevfileSelected(meta: che.DevfileMetaData): void {
+  private onDevfileClear(): void {
```

Comment:
I'd like to rename this method `onDevfileClear` -> `handleDevfileClear`, wdyt?

Reply:
done